portfolio/src/main/webapp/index.html

```diff
@@ -17,6 +17,7 @@ <h1>Orpaz's Portfolio</h1>
       <p>Click <a href="randomBook.html">here</a> to get random fantasy book</p>
       <p>Click here to get a random fact about me:</p>
       <button onclick="addRandomFact()">Give me a fact</button>
+      <button onclick="getRandomFact()">Give me a fact</button>
```

Comment:
Why add this as a new button? You can just replace the old one immediately - if you want to get it back, it's in source control.